packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -526,6 +546,63 @@ contract Attestations is IAttestations, Ownable, Initializable, UsingRegistry, R
     return accounts[account].dataEncryptionKey;
   }
 
+  /**
+   * @notice Setter of the authorized attestor for an account
+   * @param authorizedAttestor The address of the attestor to set for the account
+   * @param v The recovery id of the incoming ECDSA signature.
+   * @param r Output value r of the ECDSA signature.
+   * @param s Output value s of the ECDSA signature.
+   * @dev v, r, s constitute `authorizedAttestor`'s signature on `msg.sender`.
+   */
+  function setAuthorizedAttestor(
+    address authorizedAttestor,
+    uint8 v,
+    bytes32 r,
+    bytes32 s
+  )
+    public
+  {
+    address signer = Signatures.getSignerOfAddress(msg.sender, v, r, s);
+    require(signer == authorizedAttestor, "Incorrect signature");
+
+    accounts[msg.sender].authorizedAttestor = authorizedAttestor;
+    authorizedAttestors[authorizedAttestor] = msg.sender;
+    emit AccountAuthorizedAttestorSet(msg.sender, authorizedAttestor);
+  }
+
+  /**
+   * @notice Getter for the authorized attestor for an account
+   * @param account The address of the account to get the authorized attestor for
+   * @return The address of the authorized attestor
+   */
+  function getAuthorizedAttestor(address account) external view returns (address) {
+    return accounts[account].authorizedAttestor;
```

Comment:
What happens if an account hasn't authorized an attestor? Do we require that they do? If we follow the pattern from lockedGold, this would be named `getAttestorFromAccount` and would return the account in the event that an attestor was not authorized